### ISSUE TITLE
ENOENT when running in a container where node is not on PATH

### ISSUE THREAD

**Issue by ksperling-apple** [Wandalen/wretry.action#83]
When running in a container where node is not available on PATH, spawning the underlying action fails with ENOENT. The action should probably be spawned with the currently running node binary, instead of relying on node being on PATH.

Error log:

```
= Message of Error#1
   spawn node ENOENT
   Error starting the process
       Exec path : node /__w/_actions/Wandalen/wretry.action/v1.0.36_js_action/src/Runner.js /__w/_actions/Wandalen/checkout/dist/index.js
       Current path : /__w/redacted/redacted
= Beautified calls stack
   at Process.ChildProcess._handle.onexit (node:internal/child_process:283:19)
   at onErrorNT (node:internal/child_process:478:16)
   at processTicksAndRejections (node:internal/process/task_queues:83:21)
   
= Throws stack
   thrown at ChildProcess.handleError @ /__w/_actions/Wandalen/wretry.action/v1.0.36_js_action/node_modules/Joined.s:221908:13
   thrown at ChildProcess.handleError @ /__w/_actions/Wandalen/wretry.action/v1.0.36_js_action/node_modules/Joined.s:221932:12
= Exposed
   errno : -2
   code : 'ENOENT'
   syscall : 'spawn node'
   path : 'node'
   spawnargs : [
     '/__w/_actions/Wand...ion/src/Runner.js', 
     '/__w/_actions/Wand...out/dist/index.js'
   ]
```

**Comment by dmvict** [comment]
Hello, @ksperling-apple  
Thank you for the issue. I think your thoughts have sense. 

I still have no time to implement solution.

**Comment by dmvict** [comment]
Hello, @ksperling-apple 

The new version is released and it has the fix. Please, try it and give me a feedback.

**Comment by ksperling-apple** [comment]
I can confirm this is fixed in 1.3.0, thanks!
